### PR TITLE
Fix spelling in migration guides

### PR DESCRIPTION
**Pull request by Eric-Arellano**

### CODE REVIEW

File: scripts/config/cspell/cSpell.json [hunk 1]
```diff
@@ -45,9 +45,7 @@
     "</?thead>",
     "</?iframe",
     // Method use, e.g. Class.foo()
-    "([a-zA-Z_]\\w*)\\.[a-zA-Z_]\\w*",
```
Comment (Eric-Arellano) [inline scripts/config/cspell/cSpell.json:48]: The new regex now matches multiple periods

File: scripts/config/cspell/cSpell.json [hunk 2]
```diff
@@ -45,9 +45,7 @@
...
-    // Observable labels like IIZII
-    "[XIZ]{3,6}",
```
Comment (Eric-Arellano) [inline scripts/config/cspell/cSpell.json:50]: Turns out this was explaining the `accum` vs `accumulating` thing, somehow 🤷 Removing this fixes it

Comment (frankharkins) [reply scripts/config/cspell/cSpell.json:50]: Turns out cspell is [case insensitive by default](https://cspell.org/docs/case-sensitive/) which is why it was matching "izing" words. No idea why it matches "accumulating" though 🤔 

Either way, I think better to explicitly ignore the observable labels per page.

File: docs/api/migration-guides/qiskit-opflow-module.mdx [hunk 3]
```diff
@@ -3,6 +3,8 @@ title: qiskit.opflow migration guide
 description: How to update your code to stop using the deprecated `qiskit.opflow` module.
 ---
 
+{/* cspell:ignore XYZY, SLSQP */}
```
Comment (frankharkins) [inline docs/api/migration-guides/qiskit-opflow-module.mdx:6]:
```suggestion
{/* cspell:ignore XYZY */}
```

Nit: You ignore this in `cSpell.json`.

**Review by frankharkins — APPROVED** [review]
Nice! Impressed there was only one actual spelling correction

**Pull request merged**